tests/factories.py

```diff
@@ -0,0 +1,24 @@
+from importlib import import_module
+
+from django.conf import settings
+
+import factory
+import factory.fuzzy
+from sessionprofile.models import SessionProfile
+
+
+class SessionProfileFactory(factory.django.DjangoModelFactory):
+
+    session_key = factory.fuzzy.FuzzyText(length=40)
+
+    class Meta:
+        model = SessionProfile
+
+    @classmethod
+    def create(cls, **kwargs):
+        SessionStore = import_module(settings.SESSION_ENGINE).SessionStore
```

Comment:
This would be a nice candidate to make a separate function, as it is also used in the admin class

Reply:
One of these is a test factory, so I don't think it really warrants one

Reply:
Done